axum/src/extract/rejection.rs

```diff
@@ -104,6 +112,7 @@ composite_rejection! {
     pub enum FormRejection {
         InvalidFormContentType,
         FailedToDeserializeForm,
```

Comment:
This should have a `Query` prefix then in the next breaking-change release, I guess?

Reply:
Yes I think so!